.cirrus.yml

```diff
@@ -54,6 +54,15 @@ global_task_template: &GLOBAL_TASK_TEMPLATE
 #  install_script:
 #    - choco install python --version=3.7.7 -y
 
+lint_task:
+  name: 'lint'
+  skip: $CIRRUS_REPO_FULL_NAME == "bitcoin-core/gui" && $CIRRUS_PR == ""  # No need to run on the read-only mirror, unless it is a PR.
+  container:
+    image: ubuntu:bionic
+  pip_cache:
+    folder: /tmp/pip_cache_dir
```

Comment:
Is this really needed? I somehow supspect that setting up the cache and fetching from it is slower than fetching from the pip servers directly. Or at the least, it is not measurable and we don't care about a few milliseconds here or there anyway.